docs/src/cnpg-plugin.md

```diff
@@ -845,3 +845,43 @@ postgres=# \q
 
 This command will start `kubectl exec`, and the `kubectl` executable must be
 reachable in your `PATH` variable to correctly work.
+
+### Snapshotting a CloudNativePG cluster
+
+The `kubectl cnpg snapshot` creates snapshots of a Cluster by:
+
+1. choosing a replica Pod to work on
+2. fence the replica
+3. taking the snapshot
+4. unfence the replica
+
+!!! Important
+    A cluster having fenced instances cannot be snapshotted.
```

Comment:
I'm not 100% sure this is the right behavior, but I'd prefer to snapshot replicas belonging to healthy clusters.
If you have a fenced instances, you may be performing a manual maintenance activity, and this may not be a good moment to take a snapshot.

Not sure I got your point @fcanovai

Reply:
```suggestion
!!! Important
    A cluster already having fenced instances cannot be snapshotted.
```